tox.ini

```diff
@@ -16,12 +16,13 @@ deps =
    isort
    mccabe
    pytest
+   pytest-xdist
 
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 
 commands =
-    python -Wi {envsitepackagesdir}/coverage run -m pytest {envsitepackagesdir}/pylint/test/ {posargs:}
+    python -Wi {envsitepackagesdir}/coverage run -m pytest -n auto {envsitepackagesdir}/pylint/test/ {posargs:}
```

Comment:
Is there any good reason to keep this feature disabled by default? Manual runs by developers and contributors and done much more often than automated Travis runs.